src/__mock__/fetchMock.ts

```diff
@@ -137,3 +138,39 @@ export const mockCheckIdentifier = () => {
     }
   )
 }
+
+export const mockGetSecondaryPersons = (
```

Comment:
If you update the interface to a generic, this could be transformed into a generic function.
```suggestion
export const mockRequest<T> = (response: T[]) => {
```